inscriptions/ISic3190.xml

```diff
@@ -89,7 +93,7 @@
                                 <condition/>
 		    </supportDesc>
                             <layoutDesc>
-                                <layout><!--Default value and requires checking-->
+                                <layout><p>The inscription is located in the upper, side and lower border (it is written in a clockwise direction) of two large tiles closing a large loculus. On the right tile there is a graffito of a chi-rho monogram and a chi that may have been sketched before scratching the larger one.</p>
```

Comment:
NB, not quite clear in English - the inscription is written along the upper, right and lower edges of the loculus rim, which is covered by two large tiles, on the surface of the right hand of which is a chi-rho and an X